conda-store-server/tests/test_server.py

```diff
@@ -335,6 +336,60 @@ def test_create_specification_unauth(testclient):
     assert r.status == schema.APIStatus.ERROR
 
 
+# Only testing size values that will always cause errors. Smaller values could
+# cause errors as well, but would be flaky since the test conda-store state
+# directory might have different lengths on different systems, for instance,
+# due to different username lengths.
+@pytest.mark.parametrize(
+    "size",
+    [
+        # OSError: [Errno 36] File name too long
+        255,
+        # OSError: [Errno 36] File name too long
```

Comment:
This kind of message is Unix specific, right? How would they look on Windows?

Reply:
This is with LongPathsEnabled = 1, otherwise Windows has the MAX_PATH limit anyway.

```py
>>> open(f"c:\{'a' * 252}.txt", 'w').write('test\n')
...
OSError: [Errno 22] Invalid argument: 'c:\\aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.txt'
```

I wouldn't rely on these messages being the same between Python versions either. This is just to have some intuition why things fail. Since this is a test, I don't think the message matters much.